content/terraform-docs-common/redirects.jsonc

```
@@ -761,5 +735,596 @@
     "source": "/terraform/language/import/bulk",
     "destination": "/terraform/language/v1.14.x/import/bulk",
     "permanent": true
+  },
+  {
+    "source": "/terraform/intro/v:version/index.html",
+    "destination": "/terraform/intro/",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/intro/index.html",
+    "destination": "/terraform/intro/",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cdktf/concepts/providers-and-resources",
+    "destination": "/terraform/cdktf/concepts/providers",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cdktf/examples",
+    "destination": "/terraform/cdktf/examples-and-guides/examples",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cdktf/v:version(0\\.12\\.x)/providers",
+    "destination": "/terraform/cdktf/v0.12.x/concepts/providers",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cli/index.html",
+    "destination": "/terraform/cli/",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cli/v:version/index.html",
+    "destination": "/terraform/cli/",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/language/v:version(1\\.(?:1|2|3|4|5|6|7|8|9|10|11)\\.x)/state/remove",
+    "destination": "/terraform/cli/:version/commands/state/rm",
+    "permanent": true
```

```suggestion
    "source": "/terraform/language/v:version(v1\\.(?:1|2|3|4|5|6|7|8|9|10|11)\\.x)/state/remove",
    "destination": "/terraform/cli/:version/commands/state/rm",
    "permanent": true
```
Right? When the site switches to the destination, it is leaving out the "v" part of the version string.